examples/kafka-service-ssl/kafka_service_ssl.bal

```diff
@@ -26,7 +26,7 @@ listener kafka:Listener orderListener = new ("localhost:9094", {
 service on orderListener {
 
     remote function onConsumerRecord(Order[] orders) returns error? {
```

Comment:
Do we need `error?` return type. Looks like this query expression does not return any error.

Reply:
updated